src/overlords/core/worker.ts

```diff
@@ -39,11 +41,11 @@ export class WorkerOverlord extends Overlord {
 			4       : 5e+4,
 			5       : 1e+5,
 			6       : 5e+5,
-			7       : 1e+6,
-			8       : 2e+7,
+			7       : 2e+6,
+			8       : 2.1e+7,
```

Comment:
Because it means that it's just barely over 2 direct nukes or 4 indirect. Had some structures that were getting blown up by an extra 20k damage that really didn't need to die.